quarkchain/cluster/cluster_config.py

```diff
@@ -455,7 +461,12 @@ def from_dict(cls, d):
         config.QUARKCHAIN = QuarkChainConfig.from_dict(config.QUARKCHAIN)
         config.MONITORING = MonitoringConfig.from_dict(config.MONITORING)
         config.MASTER = MasterConfig.from_dict(config.MASTER)
-        config.SLAVE_LIST = [SlaveConfig.from_dict(s) for s in config.SLAVE_LIST]
+        config.SLAVE_LIST = [
+            SlaveConfig.from_dict(s)
+            for s in config.SLAVE_LIST
+            if SlaveConfig.from_dict(s).TYPE == "QKCRPC"
+        ]
+        config.GRPC_SLAVE_LIST = [s for s in config.SLAVE_LIST if s.TYPE == "GRPC"]
```

Comment:
`config.SLAVE_LIST` already filtered, this seems wrong. please add a unit test for the config to make sure